libs/shared/components/src/auth/UserMenu/UserProfileInfo.tsx

```diff
@@ -1,66 +1,34 @@
+import { Box, Text } from '@island.is/island-ui/core'
 import { useLocale } from '@island.is/localization'
-import { parsePhoneNumberFromString } from 'libphonenumber-js'
-import { sharedMessages } from '@island.is/shared/translations'
-import { useGetUserProfileQuery } from '../../../gen/graphql'
-import * as styles from './UserMenu.css'
+import { UserDropdownItem } from './UserDropdownItem'
+import { ServicePortalPath, m } from '@island.is/service-portal/core'
 
-export const UserProfileInfo = () => {
-  const { value: showPersonalInfo } = useFeatureFlag(
-    'isServicePortalPersonalInformationModuleEnabled',
-    false,
-  )
-  const { data } = useGetUserProfileQuery({
-    skip: !showPersonalInfo,
-    fetchPolicy: 'cache-and-network',
-  })
+export const UserProfileInfo = ({ onClick }: { onClick: () => void }) => {
   const { formatMessage } = useLocale()
-  if (showPersonalInfo) {
-    const settings = data?.getUserProfile
 
-    const phoneNumber = parsePhoneNumberFromString(
-      settings?.mobilePhoneNumber || '',
-      'IS',
-    )
-    return (
-      <>
-        {settings?.email && (
-          <Box
-            display="flex"
-            alignItems="center"
-            marginBottom={1}
-            className={styles.breakWord}
-          >
-            <Box display="flex" alignItems="center" marginRight={2}>
-              <Icon type="outline" icon="mail" color="blue300" />
-            </Box>
-            <Text>{settings.email}</Text>
-          </Box>
-        )}
-        {phoneNumber?.isValid() && (
-          <Box
-            display="flex"
-            alignItems="center"
-            marginBottom={2}
-            className={styles.breakWord}
-          >
-            <Box display="flex" alignItems="center" marginRight={2}>
-              <Icon type="outline" icon="call" color="blue300" />
-            </Box>
-            <Text>{phoneNumber.nationalNumber}</Text>
-          </Box>
-        )}
-        {(settings?.email || settings?.mobilePhoneNumber) && (
-          <a href="/minarsidur/stillingar/personuupplysingar">
-            <Button variant="text" icon="arrowForward" size="small">
-              {formatMessage(sharedMessages.settings)}
-            </Button>
-          </a>
-        )}
-      </>
-    )
-  }
-  return null
+  return (
+    <Box>
+      <Box marginBottom={1}>
+        <Text variant="small">{formatMessage(m.settings)}</Text>
+      </Box>
+
+      <Box>
+        <UserDropdownItem
+          text={formatMessage(m.personalInformation)}
+          link={ServicePortalPath.SettingsPersonalInformation}
+          icon={{ type: 'outline', icon: 'person' }}
+          onClick={() => onClick()}
+        />
+      </Box>
+      <Box paddingTop={1}>
+        <UserDropdownItem
+          text={formatMessage(m.accessControl)}
+          link={ServicePortalPath.SettingsAccessControl}
+          icon={{ type: 'outline', icon: 'people' }}
+          onClick={() => onClick()}
+        />
+      </Box>
```

Comment:
These links don't work in other apps (application-system-form) since they use `react-router-dom` links relative to `/minarsidur`. May need to use normal `<a>` links with full path, at least in other apps.

Reply:
Can test here: https://service-portalnew-design-demo-beta.dev01.devland.is/umsoknir/example

Reply:
Ah - I'll fix, thanks

Reply:
Done ✅